react/components/ShippingSimulator/components/ShippingTableRow.js

```diff
@@ -3,29 +3,29 @@ import PropTypes from 'prop-types'
 import { intlShape, injectIntl } from 'react-intl'
 import classNames from 'classnames'
 
-const ShippingTableRow = ({ name, eta, value, intl }) => {
+const ShippingTableRow = ({ name, shippingEstimate, price, intl }) => {
   const etaClassName = classNames('vtex-shipping-table__cell', {
-    'vtex-shipping-table__cell--center': eta === undefined,
+    'vtex-shipping-table__cell--center': shippingEstimate === undefined,
   })
 
   const valueClassName = classNames('vtex-shipping-table__cell', {
-    'vtex-shipping-table__cell--center': value === undefined,
+    'vtex-shipping-table__cell--center': price === undefined,
   })
 
   let etaText, valueText
 
-  if (eta === undefined) {
+  if (shippingEstimate === undefined) {
     etaText = '-'
   } else {
-    etaText = intl.formatMessage({ id: 'shipping.eta' }, { eta })
+    etaText = intl.formatMessage({ id: 'shipping.eta' }, { eta: shippingEstimate })
   }
 
-  if (value === undefined) {
+  if (price === undefined) {
     valueText = '-'
-  } else if (value === 0) {
+  } else if (price === 0) {
     valueText = intl.formatMessage({ id: 'shipping.free' })
   } else {
-    valueText = intl.formatNumber(value, { style: 'currency', currency: 'BRL' })
+    valueText = intl.formatNumber(price, { style: 'currency', currency: 'BRL' })
```

Comment:
Is this always BRL, or should be according to user's location?

Reply:
I believe so